### PR TITLE
[44] Add Sign in and Sign up UI

### PR DESCRIPTION
**Pull request by bongudth**

### CODE REVIEW

File: src/resources/public/css/style.css [hunk 1]
```diff
@@ -43,7 +43,8 @@ body {
   height: 70px;
 }
 
-#app-button-sign-in {
+#app-button-signin,
+#app-button-shorten {
```
Comment (bittermeatball) [inline src/resources/public/css/style.css:47]: Use CSS class instead

File: src/resources/public/css/style.css [hunk 2]
```diff
@@ -103,6 +96,34 @@ body {
...
+}
+
+.eye-signup-1 {
+  top: 12.3rem;
+}
+
+.eye-signup-2 {
+  top: 16.9rem;
+}
+
```
Comment (bittermeatball) [inline src/resources/public/css/style.css:126]: Do not use number to mark element, specify element functionality in class name like:
```
.signup-button-icon
.signup-button-icon-left
...
```

File: src/resources/views/signin.ejs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          <input type="password" id="app-input-password" class="app-input password" placeholder="Input your password" />
+          <i id="eye-password" class="eye-password eye-signin far fa-eye-slash"></i>
+        </div>
+        <div class="app-remember-forgot">
+          <div class="app-remember-container">
+            <input type="checkbox" class="checkbox-remember"/>
+            <span>Remember me</span>
+          </div>
+          <a href="./forgot.ejs" class="forgot-password">Forgot your password?</a>
+        </div>
```
Comment (bittermeatball) [inline src/resources/views/signin.ejs:41]: Is the `.ejs` extension required?

File: src/resources/views/signup.ejs [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      <div class="loader">
+        <div class="dot dot1"></div>
+        <div class="dot dot2"></div>
+        <div class="dot dot3"></div>
+        <div class="dot dot4"></div>
+        <div class="dot dot5"></div>
+        <div class="dot dot6"></div>
+        <div class="dot dot7"></div>
+      </div>
+    </div>
```
Comment (bittermeatball) [inline src/resources/views/signup.ejs:62]: Consider use EJS partial to reuse this piece of code
See blog: https://www.digitalocean.com/community/tutorials/how-to-use-ejs-to-template-your-node-application

File: src/resources/views/signup.ejs [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-NMY3QQNKFS">
+  </script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag('js', new Date());
+
+    gtag('config', 'G-NMY3QQNKFS');
```
Comment (bittermeatball) [inline src/resources/views/signup.ejs:76]: Consider use EJS partial to reuse this piece of code too

**Review by bittermeatball — CHANGES_REQUESTED** [review]

File: src/resources/views/signin.ejs (hunk 3 above)
Comment (ichhoa129) [reply src/resources/views/signin.ejs:41]: In Ejs, the *href* attribute should only contain the route, such as  */forgot*. The *.* is redundant and the *.ejs* extension may not work.

**Review by prd-y-nguyen — APPROVED** [review]

**Review by bittermeatball — APPROVED** [review]

**Pull request merged**